lib/graphql/lang/ast/type_info_visitor.ex

```diff
@@ -127,7 +127,7 @@ defmodule GraphQL.Lang.AST.TypeInfoVisitor do
         :ObjectField ->
           input_type = TypeInfo.input_type(accumulator[:type_info])
           object_type = TypeInfo.named_type(accumulator[:type_info], input_type)
-          if object_type === %GraphQL.Type.ObjectType{} do
+          if %GraphQL.Type.ObjectType{} = object_type do
```

Comment:
this is usually a sign that a method can be extracted out and the code made a little cleaner. Then again that requires understanding the code and based on that comment I'm not sure if know what's happening :cry: